### PR TITLE
A blog post about Trends page

### PR DESCRIPTION
**Pull request by luluz66**

### CODE REVIEW

File: website/blog/building-a-fast-trends-page.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+slug: building-a-fast-trends-page.md
+title: "Building a Fast Trends Page"
+author: Lulu Zhang
+author_title: Engineer @ BuildBuddy
+date: 2022-11-11:12:00:00
+author_url: https://www.linkedin.com/in/luluzhang66
+author_image_url: https://avatars.githubusercontent.com/u/3977083?v=4
+image: /img/blog
```
Comment (luluz66) [inline website/blog/building-a-fast-trends-page.md:9]: waiting for an image from Siggi

Comment (luluz66) [reply website/blog/building-a-fast-trends-page.md]: Added Image.

File: website/blog/building-a-fast-trends-page.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+slug: building-a-fast-trends-page.md
+title: "How We Use ClickHouse to Analyze Trends Across Millions of Builds"
+author: Lulu Zhang
+author_title: Engineer @ BuildBuddy
+date: 2022-11-11:12:00:00
+author_url: https://www.linkedin.com/in/luluzhang66
+author_image_url: https://avatars.githubusercontent.com/u/3977083?v=4
+image: /img/blog/clickhouse.png
```
[the remark this answers is not in the document]
Comment (siggisim) [reply website/blog/building-a-fast-trends-page.md:9]: i think i sent @luluz66 an updated image without that part - lulu lmk if you need me to send it

[the remark this answers is not in the document]
Comment (luluz66) [reply website/blog/building-a-fast-trends-page.md:9]: I updated the image

File: website/blog/building-a-fast-trends-page.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+![](/img/blog/trends-2.png)
+
+The trends page allows you to see how improvements you are making to your builds
+affects your average build duration and other stats. It also exposes areas that
+might need improving. For example, if you see the cache hit rate go down over
+time, your build might have some non-deterministic build actions that could be
+improved, or some newly introduced dependencies in your build that result in
+more frequent cache invalidations.
+
+When we first created the Trends page, MySQL queries were used to aggregate
```
[the remark this answers is not in the document]
Comment (luluz66) [reply website/blog/building-a-fast-trends-page.md]: Done

**Review by bduffany — APPROVED** [review]

File: website/blog/building-a-fast-trends-page.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+a [Trends](https://app.buildbuddy.io/trends/) page to visualize trends in this
+data over time.
+
+![](/img/blog/trends-2.png)
+
+The trends page allows you to see how improvements you are making to your builds
+affects your average build duration and other stats. It also exposes areas that
+might need improving. For example, if you see the cache hit rate go down over
+time, your build might have some non-deterministic build actions that could be
+improved, or some newly introduced dependencies in your build that result in
```
Comment (tylerwilliams) [inline website/blog/building-a-fast-trends-page.md:25]: nit: I think you can remove `in your build`

`your build might have some` covers the second clause just fine

**Review by tylerwilliams — APPROVED** [review]
LGTM

File: website/blog/building-a-fast-trends-page.md (hunk 4 above)
Comment (luluz66) [reply website/blog/building-a-fast-trends-page.md]: Done

**Pull request merged**